### PR TITLE
[NETBEANS-3733] Suggest surefire 2.22.0 for JUnit5 tests.

### PR DESCRIPTION
**Pull request by sdedic**
Small improvement to 073c42c1cd: the IDE properly tests for JUnit5 in `runSingleMethodEnabled`, so it triggers a suggest/status line message if the project is not properly set up. But the display message and POM fix installs JUnit4+older surefire, which does not work in JUnit5 environment.

I've just improved the messages for JUnit5 case + used proper Surefire version.

### CODE REVIEW

File: java/maven/src/org/netbeans/modules/maven/ActionProviderImpl.java [hunk 1]
```diff
@@ -304,9 +309,24 @@ private boolean checkSurefire(final String action) {
                         RequestProcessor.getDefault().post(new Runnable() {
                             @Override
                             public void run() {
+                                String surefireVersion = null;
+                                String junitVersion = null;
+                                
+                                if (ju5 && !usingSurefire2_22()) {
+                                    surefireVersion = SUREFIRE_VERSION_SAFE_5;
+                                } else if (usingSurefire28()) {
```
Comment (jlahoda) [inline java/maven/src/org/netbeans/modules/maven/ActionProviderImpl.java:317]: Not sure about the logic here - should it be:
```suggestion
                                } else if (!usingSurefire28()) {
```

Comment (sdedic) [reply java/maven/src/org/netbeans/modules/maven/ActionProviderImpl.java:317]: Typo - thanks !

Comment (sdedic) [reply java/maven/src/org/netbeans/modules/maven/ActionProviderImpl.java]: Works much better ... in 78778ee

**Comment by mcdonnell-john** [comment]
Probably too late for 12.0, so lets put a milestone of 12.1 on it so its not missed later on

**Comment by neilcsmith-net** [comment]
@mcdonnell-john not sure, it's certainly an annoyance.  There's a chance we're going to need one more beta, so putting 12.0 on it and leaving it to @ebarboni to make the call.

**Comment by neilcsmith-net** [comment]
OTOH, @sdedic seems to have gone for it anyway!

**Comment by sdedic** [comment]
OOPS! Feel free to rollback :-/

**Comment by neilcsmith-net** [comment]
@sdedic :laughing: well Eric really will need to review it now!

**Pull request merged**